builder/arm64/README.md

```diff
@@ -0,0 +1,27 @@
+# filecoin-ffi build workaround
+
+As of this writing, .circleci does not support arm based images and as a workaround we need to try and build the filecoin-ffi in a containerized environment. The workaround is to create a docker image using an ARM64 based operating system and build the binaries in the image. We can then pull (`cp`) the binaries to the host.
```

Comment:
I'm not sure what "arm based images" are, but CircleCI has ARM64 support, which is e.g. be used by rust-fil-proofs: https://github.com/filecoin-project/rust-fil-proofs/blob/6260c2a5409bda6c2514e8c355d52623939f4bc7/.circleci/config.yml#L19-L22

Reply:
Ok. Thank you for this. I'll experiment with this image then.

I saw this note on circle ci.
![image](https://user-images.githubusercontent.com/4479171/167905828-b6e7a375-b653-4eca-8415-0cb9b14ecbcb.png)